docs/source/sharing.rst

```diff
@@ -0,0 +1,93 @@
+Sharing
```

Comment:
Yes, the discussion came up because "sharing" means 'send to social media, or email, or whatever' these days, and therefore is confusing. "Access" would be a better term, but I am not quite sure what is the right way to implement that. Does it need a PLIP?
Probably since it's a change that needs to be reflected also in documentation, and that at least would make it visible in changelogs, release notes and the like. 

But I've discussed it only from a user/documentation standpoint, and actually haven't looked at Plone codebase if the change would lead to confusion there and therefore should only be done towards 6, or if it's a label-only thing that can be changed in a minor release.